asn1crypto/cms.py

```diff
@@ -419,7 +419,10 @@ class RecipientIdentifier(Choice):
 
 class KeyEncryptionAlgorithmId(ObjectIdentifier):
     _map = {
-        '1.2.840.113549.1.1.1': 'rsa'
+        '1.2.840.113549.1.1.1': 'rsa',
+        '2.16.840.1.101.3.4.1.5': 'aes128-wrap',
+        '2.16.840.1.101.3.4.1.25': 'aes192-wrap',
+        '2.16.840.1.101.3.4.1.45': 'aes256-wrap',
     }
```

Comment:
For consistency with the other `asn1crypto` OID names, can we use `aes128_wrap`, etc?